examples/rekey-example/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import {DocumentMetadata, TenantSecurityClient} from "@ironcorelabs/tenant-security-nodejs/";
+import * as fs from "fs";
```

Comment:
Yeah, it seems to be. Do you know why I wouldn't have gotten that warning when I ran the example locally>

Reply:
There probably is a way to run it locally that would give you that warning. If you turn on `noUnusedLocals` + `noUnusedParameters` I think it would've been a compilation error.